src/components/Snackbar/CustomSnackbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Snackbar, Alert } from '@mui/material';
+
+interface SnackbarProps {
+  res: string;
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export const CustomSnackbar = ({ res, open, setOpen }: SnackbarProps) => {
+  const handleClose = (event: React.SyntheticEvent | Event, reason?: string) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Snackbar
+        anchorOrigin={{
+          vertical: 'top',
+          horizontal: 'center',
+        }}
+        open={open}
+        autoHideDuration={6000}
+        onClose={handleClose}
+        message="Note archived"
+      >
+        <Alert onClose={handleClose} severity="success" sx={{ width: '100%' }}>
```

Comment:
Should assign severity = { variable } not always using success type. Implement AlertColor from MUI

Reply:
![image](https://user-images.githubusercontent.com/64080330/165008071-b265f3ff-19bb-4bf2-849d-346c455f10cc.png)